--- ChangeLog	2026-05-07 10:28:24.150100728 -0400
+++ ChangeLog-spell-corrected	2026-05-07 10:28:55.271140147 -0400
@@ -8176,7 +8176,7 @@
 
 2004-06-27  R. Bernstein <rocky@gnu.org>
 
-	* doc/Makefile.am: Don't distrubute makedb.txt any more.
+	* doc/Makefile.am: Don't distribute makedb.txt any more.
 
 2004-06-27  R. Bernstein <rocky@gnu.org>
 
@@ -8210,7 +8210,7 @@
 
 2004-06-26  R. Bernstein <rocky@gnu.org>
 
-	* src/job.c: Go into the debugger when we recieve a signal.
+	* src/job.c: Go into the debugger when we receive a signal.
 
 2004-06-26  R. Bernstein <rocky@gnu.org>
 
@@ -8338,7 +8338,7 @@
 2004-06-13  R. Bernstein <rocky@gnu.org>
 
 	* src/dbg_cmd.c: Expand the way set works.  "set *var* *val*" is now
-	"set variable *var* *val*" so we can accomodate set basename {on|off|toggle| set trace {on|off|toggle}
+	"set variable *var* *val*" so we can accommodate set basename {on|off|toggle| set trace {on|off|toggle}
 
 2004-06-12  R. Bernstein <rocky@gnu.org>
 
